src/main.cpp

```diff
@@ -77,10 +78,14 @@ int main(int argc, char **argv) {
     // Set the current view frustum
     gameManager.setViewFrustum(new ViewFrustum());
 
+    // Set the Shadow Map Object
+    ShadowMap* shadowMap = new ShadowMap();
+    gameManager.setShadowMap(shadowMap);
+
     // Add all static objects before this!!!
     world.init();
 
-    gameManager.setTime(15.0);
+    gameManager.setTime(1500.0);

```

Comment:
I'm fine with increasing the time but just want to check this wasn't left over from debugging unintentionally